docs/mixins/state.md

```diff
@@ -248,6 +249,33 @@ myToolKitApp.getState('foo');
 myToolKitApp.getState();
 ```
 
+### Toggling State
+
+`StateMixin` has a `toggleState` method that sets the `StateModel` instance attribute to a boolean value.
+Attributes that do not exist on the state will be created.
+Not passing in a value will toggle the attribute's current value, while non-boolean values will be coerced to `true` or `false`.
+
+```js
+var MyToolKitApp = new Marionette.Toolkit.App({
+    StateModel: {
```

Comment:
This isn't how state model defaults works.  this is a mistake in the current documentation